src/txpool/getters.yaml

```diff
@@ -0,0 +1,69 @@
+- name: txpool_transactions
+  summary: Returns a list of all transactions that match the supplied filter conditions that are either pending for inclusion in the next block(s) or scheduled for future execution.
+  params:
+    - name: Filter
+      schema:
+        title: txpool filter
+        type: object
+        properties:
+          from:
+            title: from address
+            $ref: '#/components/schemas/address'
+          to:
+            title: to address
+            $ref: '#/components/schemas/address'
+          gas:
+            title: gas limit
+            $ref: '#/components/schemas/FilterOperators'
+          gasPrice:
+            title: gas price
+            $ref: '#/components/schemas/FilterOperators'
+          maxFeePerGas:
+            title: max fee
+            $ref: '#/components/schemas/FilterOperators'
+          maxPriorityFeePerGas:
+            title: max priority fee
+            $ref: '#/components/schemas/FilterOperators'
+          value:
+            title: value
+            $ref: '#/components/schemas/FilterOperators'
+          nonce:
+            title: nonce
+            $ref: '#/components/schemas/FilterOperators'
+  result:
+    name: Result
+    schema:
+      title: Result
+      schema:
+        $ref: '#/components/schemas/TxPoolTransactions'
+        title: transactions
+        type: object
+        properties:
+          pending:
+            title: pending
+            type: array
+            items:
+              $ref: '#/components/schemas/GenericTransaction'
+          queued:
+            title: queued
+            type: array
+            items:
+              $ref: '#/components/schemas/GenericTransaction'
+- name: txpool_statistics
+  summary: Returns statistics about the node's transaction pool.
+  params: []
+  result:
+    name: Statistics
+    schema:
+      title: statistics
+      type: object
+      properties:
+        maxSize:
```

Comment:
This probably needs more elaboration, since depending on implementations there could not be the concept of max size, intended as a fixed limit on the number of transactions that can fit in the txpool. 
For example the new layered txpool implementation in Besu, there is not a fixed limit in terms of number of transactions, but the limit is the amount of estimated memory used by transactions, this to better adapt to scenarios where there could be very large txs like for EIP-4844.
Other implementations could also extend the txpool to the disk, again allowing for different kind of limits that are based on the amount of space, more than a fixed number.

My first suggestion is to simply remove this field, but if there are valid use cases for having it, then we can elaborate it, and for example we can see if providing a percentage of how much the txpool is filled can be a better option

Reply:
Interesting. Altho I don't think 4844 transactions will be included in the same tx pool (at least not in Geth), the layered pool sounds interesting. 

And removing the maxSize property

Reply:
I think that from the RPC APIs point of view, there should not be any distinction between tx types, and so blob txs should be treated as the other and returned when present, letting any client free to choose how to implement the txpool.